### ISSUE TITLE
Can't find declaration file in 0.5.0

### ISSUE THREAD

**Issue by aydrian** [kiliman/remix-params-helper#40]
Receiving the following TypeScript error after upgrading to 0.5.0
```
Could not find a declaration file for module 'remix-params-helper'. 
```
I am using Remix 2.0 with Vite and ESM modules.
If I go back to 0.4.10, the problem goes away.

**Comment by kiliman** [comment]
That's odd. I tested on both Remix v1.9+ and v2.2 with Vite. 

I'll try again with a fresh project.

**Comment by kiliman** [comment]
I created a test repo using Vite, and it's working fine.

https://github.com/kiliman/remix-vite-params

[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/~/github.com/kiliman/remix-vite-params)

**Comment by aydrian** [comment]
Let me see if there's any other differences using your test repo and try again.

**Comment by aydrian** [comment]
Weird. I'm still seeing the type error and I don't see anything different. tsconfig.json looks fine. The vite.config looks fine. Here's a route where I'm using it: https://github.com/aydrian/itsaydrian-stream-remix/blob/main/app/routes/scenes%2B/me%2B/_frame%2B/chatting%2B/%24num.tsx

**Comment by kiliman** [comment]
Ah, I now see what you mean. It's a VS Code warning. I thought you were seeing an error during build or runtime.

<img width="1036" alt="image" src="https://github.com/kiliman/remix-params-helper/assets/47168/d36f460d-7e54-435d-871d-fbc82cfb1319">

I'll take a look. Thanks.

**Comment by aydrian** [comment]
Sorry. It also shows up when I run `npm run typecheck`

**Comment by kiliman** [comment]
Fixed in v0.5.1

**Comment by nicolas-besnard** [comment]
I have the same issue (with version `0.5.1`) using the indie-stack with `remix 1.16.0`. Same as Aydrian, it works well in `0.4.1`
